core/src/androidTest/java/com/amplifyframework/core/DevMenuMainViewInstrumentationTest.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core;
+
+import androidx.fragment.app.testing.FragmentScenario;
+import androidx.navigation.NavDestination;
+import androidx.navigation.Navigation;
+import androidx.navigation.testing.TestNavHostController;
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.espresso.action.ViewActions;
+import androidx.test.espresso.matcher.ViewMatchers;
+
+import com.amplifyframework.testutils.Sleep;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import static androidx.test.espresso.Espresso.onView;
+
+/**
+ * Tests the navigation behavior for {@link DevMenuMainFragment}.
+ */
+public final class DevMenuMainViewInstrumentationTest {
+    // Amount of time (in milliseconds) to wait before starting
+    // a test, to allow any animations to complete.
+    private static final int START_TEST_DELAY_MS = 500;
+    // A navigation host controller for testing.
+    private static TestNavHostController navHostController;
```

Comment:
does this variable have to be static?

Reply:
No. I think that may have been left over from a previous iteration of these tests, so I'll remove "static".